content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -632,7 +632,7 @@ Create that Pod, by running:
 kubectl create -f https://kubernetes.io/examples/pods/pod-configmap-env-var-valueFrom.yaml
 ```
 
-That pod produces the following output from the `test-container` container:
+That pod produces the following output from the `test-container` container (run `kubectl logs dapi-test-pod`):
```

Comment:
@nirajchandak,  Please remove the command form here and Mention this command in the next line under the shell code block snippet.

Reply:
@nirajchandak
Please address the above comment.

Reply:
sure 👍